sdk-extensions/incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/trace/zpages/ZPageServer.java

```diff
@@ -183,6 +188,20 @@ private static void stop() {
     }
   }
 
+  /**
+   * Convenience method to return a new SdkTracerProvider that has been configured with our ZPage
+   * specific span processor, sampler, and limits.
+   *
+   * @return new SdkTracerProvider
+   */
+  public SdkTracerProvider buildSdkTracerProvider() {
+    return SdkTracerProvider.builder()
+        .addSpanProcessor(getSpanProcessor())
+        .setSpanLimits(getTracezTraceConfigSupplier())
+        .setSampler(getTracezSampler())
+        .build();
+  }
```

Comment:
I would have this accept a `SdkTracerProviderBuilder` as an argument to make it more flexible. 

Also, can we add test coverage for this method?